src/specs/models/samples-bodies/BeginSampleFileUpload.v2.yaml

```diff
@@ -15,7 +13,6 @@ properties:
           - pattern: v3
 
 required:
-  - size
```

Comment:
Why is size no longer added to sql?

Reply:
In this endpoint we gave `sampleFile#beginUpload` the size because it needed to know the true size of the file to calculate how many signedUrls (how many parts) the upload required. Now this information is not needed because we give signedUrls when they are requested

Reply:
The size already in sql when a sample was created didn't help in this endpoint if files needed to be compressed because it was a size calculated based on the original file, not the one that is uploaded post compression